### DISCUSSION TITLE
problem with net framework 4.6.1

### DISCUSSION THREAD

**Discussion by CrhistopherDiaz** [jchristn/WatsonTcp#253] [Q&A] [answered]
Problem with net framework 4.6.1 watsontcp 5 gave compatibility to the version and I thank you from the bottom of my heart but it cannot be installed for use with that version since the dependency that uses System.Text.Json needs mandatory use in its version 7 net framework 4.6.2 could you do something to correct this please and thanks for your attention

**Comment by jchristn** [comment, marked as answer]
In that case you'll have to clone source, remove the dependency, and add something like ```Newtonsoft.Json``` and implement your own serialization helper.
